addon/components/detail-instructors-list.hbs

```diff
@@ -9,7 +10,29 @@
     <ul class="instructors-list" data-test-instructors>
       {{#each (sort-by "fullName" @instructors) as |user|}}
         <li>
-          {{user.fullName}}
+          <span data-test-name>
```

Comment:
This chunk of template seems to be repeated a lot. The only difference I can spot is that `data-test` attribute on the full name span. Wouldn't making this a component simplify the mouse over registration for the tooltip as well as removing duplicate template and JS code?

Reply:
all switched out and consolidated. i did not experience any of the performance degradation i was worried about.